providers/src/airflow/providers/amazon/aws/transfers/google_api_to_s3.py

```diff
@@ -175,19 +174,15 @@ def _load_data_to_s3(self, data: dict) -> None:
             replace=self.s3_overwrite,
         )
 
-    def _update_google_api_endpoint_params_via_xcom(
-        self, task_instance: TaskInstance | TaskInstancePydantic
-    ) -> None:
+    def _update_google_api_endpoint_params_via_xcom(self, task_instance: TaskInstance) -> None:
```

Comment:
As you change the signature in Google provider - will this not break compatibility of the provider with Airflow 2.10? Here I only see Type checking code... mhm, so functional should be working.

Reply:
Yeah. It's just non-functional change.